File: Load_Data_Different_Formats/script2.R
# Reading a tab-delimited text file
data_txt <- read.table("Student_Performance.txt", header = TRUE, sep = "\t")

# Reading a comma-separated file
data_csv <- read.csv("Student_Performance.csv", header = TRUE)


library(readxl)

data_xlsx <- read_excel("Student_Performance.xlsx")


text <- pdf_text("Student_Performance.pdf")

File: Load_Data_Different_Formats/script2.py

# read csv file
import pandas as pd
data_csv = pd.read_csv('Student_Performance.csv')

#read txt file
data_txt = pd.read_csv('Student_Performance.txt', delimiter='\t')

#read xlsx file from first sheet
data_xlsx = pd.read_excel('Student_Performance.xlsx', sheet_name=0)

#read pdf file
#pip install tabula-py
from tabula import read_pdf
data_pdf = read_pdf('Student_Performance.pdf')
